### ISSUE TITLE
VAMC "online scheduling" button should be styled as a link

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#21743]
## Issue Description
![image](https://user-images.githubusercontent.com/55411834/111682025-8006ef80-87e9-11eb-979c-4d1ccae550bf.png)

The blue "Schedule an appointment online" button connects the user to https://www.va.gov/health-care/schedule-view-va-appointments/. Because it is a link, it should not be styled as a CTA/button to be accessible. 

---
## Tasks
- [ ] Change "Schedule an appointment online" styling to the link design component

## Acceptance Criteria
- [ ] "Schedule an appointment online" is styled as a link. 
- [ ] The link to [Schedule and manage health appointments page](https://www.va.gov/health-care/schedule-view-va-appointments/) is maintained. 
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)
